homeassistant/components/vesync/sensor.py

```diff
@@ -252,6 +211,9 @@ def native_value(self) -> StateType:
         """Return the state of the sensor."""
         return self.entity_description.value_fn(self.device)
 
-    def update(self) -> None:
+    async def async_update(self) -> None:
         """Run the update function defined for the sensor."""
-        return self.entity_description.update_fn(self.device)
+        # I think this section can be removed since energy updates are now auto called?
+        # if self.entity_description.update_fn:
+        # return await self.entity_description.update_fn(self.device)
+        return
```

Comment:
I am actually also not sure why energy had that separate function, I think this is something we have to find out tho